auditbeat/module/file_integrity/config.go

```diff
@@ -40,14 +42,16 @@ const (
 // Config contains the configuration parameters for the file integrity
 // metricset.
 type Config struct {
-	Paths               []string   `config:"paths" validate:"required"`
-	HashTypes           []HashType `config:"hash_types"`
-	MaxFileSize         string     `config:"max_file_size"`
-	MaxFileSizeBytes    uint64     `config:",ignore"`
-	ScanAtStart         bool       `config:"scan_at_start"`
-	ScanRatePerSec      string     `config:"scan_rate_per_sec"`
-	ScanRateBytesPerSec uint64     `config:",ignore"`
-	Recursive           bool       `config:"recursive"` // Recursive enables recursive monitoring of directories.
+	Paths               []string        `config:"paths" validate:"required"`
+	HashTypes           []HashType      `config:"hash_types"`
+	MaxFileSize         string          `config:"max_file_size"`
+	MaxFileSizeBytes    uint64          `config:",ignore"`
+	ScanAtStart         bool            `config:"scan_at_start"`
+	ScanRatePerSec      string          `config:"scan_rate_per_sec"`
+	ScanRateBytesPerSec uint64          `config:",ignore"`
+	Recursive           bool            `config:"recursive"` // Recursive enables recursive monitoring of directories.
+	Exclude             []string        `config:"exclude_files"`
+	ExcludeMatchers     []match.Matcher `config:",ignore"`
```

Comment:
`match.Matcher` implements `ucfg.StringUpacker` so it will automatically be populated with the compiled regexes. If any of them fail to compile ucfg should emit an error. So I don't think you need the `[]string` version at all.